client/asset/btc/btc.go

```diff
@@ -5469,10 +5466,15 @@ func (btc *baseWallet) RefundBond(ctx context.Context, ver uint16, coinID, scrip
 }
 
 // BondsFeeBuffer suggests how much extra may be required for the transaction
-// fees part of required bond reserves when bond rotation is enabled.
-func (btc *baseWallet) BondsFeeBuffer() uint64 {
-	// 150% of the fee buffer portion of the reserves.
-	return 15 * bondsFeeBuffer(btc.segwit, btc.feeRateLimit()) / 10
+// fees part of required bond reserves when bond rotation is enabled. The
+// provided fee rate may be zero, in which case the wallet will use it's own
+// estimate or fallback value.
+func (btc *baseWallet) BondsFeeBuffer(feeRate uint64) uint64 {
+	if feeRate == 0 {
+		feeRate = btc.targetFeeRateWithFallback(1, 0)
+	}
+	feeRate *= 2 // double the current fee rate estimate so this fee buffer does not get stale too quickly
+	return bondsFeeBuffer(btc.segwit, feeRate)
```

Comment:
With `BondsFeeBuffer` based on a live rate, there's a real possibility that it could be hard to get past the wallet funding form if they refresh the page after sending the exact recommended amount, as this would force a new estimate that could be a bit higher than what they just sent.

How can we have this more robust, but also using a sensible rate for the asset?  Possibly we round the feeRate up to the nearest 10 so single digit variation won't change the result?

Reply:
Yeah, this definitely needs a solution.  I just produced the behavior I described:

`2023-06-15 17:43:58.974 [DBG] CORE[btc]: Retrieved fee rate from external API: 29`
![image](https://github.com/decred/dcrdex/assets/9373513/fa67450c-fb36-4b0d-861b-d85ae575f94b)

Then I sent that exact amount, but did **not* wait for it to confirm and proceed automatically.

Then I clicked back and then again into BTC:
`[DBG] CORE[btc]: Retrieved fee rate from external API: 39`
![image](https://github.com/decred/dcrdex/assets/9373513/c6b6b851-937c-4f42-af0f-a961a73972bb)

When my tx confirms, it won't satisfy the frontend's checks, although ReserveBondFunds probably would because of the 150% vs 200% scaling described in my previous comment.

Reply:
I'm going to have the backend (either webserver handler or core) cache the fee buffer for maybe an hour.  This value is heavily padded and meant to be good for weeks anyway.